### ISSUE TITLE
Auto ESPM import -- criteria for duplicate records not being added

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#1847]
### Expected Behavior
When using the Auto ESPM import, duplicate records from previous imports would not be imported.
### Actual Behavior 
In a very few cases, what appeared to be duplicate records were imported. It would be nice to understand under what circumstances that happens. In general, however, duplicate records are NOT imported (which is good).
### Steps to Reproduce
See doc in this issue folder 
https://drive.google.com/open?id=1bqLcV3CJil7MkaQYK2dqNg6sNy9IxnkS
### Instance Information
Server Instance: LBNL production
Server SHA: 148471a9

**Comment by RDmitchell** [comment]
I am wondering if the program is NOT importing duplicate records when the data is imported with the auto-import from ESPM. I assume it would be eliminating duplicates, but it appears that it might not be -- see the doc above for some notes I have added at the end.

**Comment by RDmitchell** [comment]
*****
Comment from Nick (in gdoc)
it does not import duplicate records that exist in the same file/import... if there are duplicates across multiple files, then I think it does import them twice.
*****
Comment from Robin (in gdoc)
I thought that the program ignored duplicate records even across different files. If not, we need to add that capability -- I will add this to the main Github issue for this

**Comment by adrian-lara** [comment]
@RDmitchell I looked into the files mentioned in the linked Google Doc. It looks like the users were correct to assume that, in the case where the `... APN` field differed between the existing and incoming records, the incoming made it through and ended up being matched and merged together because of that `... APN` difference.

In the single case where the `... APN` field was the same (empty), I'm seeing that the `Site EUI` and `Source EUI` values are different. That's what allowed for the incoming record to make it past the deduplication stage. 

Given the date when this issue was first created, I think it's worth mentioning two features that have been added that'll help users notice these differences more easily:
1. Inventory detail view - highlighting differences - addressed [here](https://github.com/SEED-platform/seed/issues/1591#issuecomment-605247762) and [here](https://github.com/SEED-platform/seed/issues/2072#issuecomment-591207979)
2. Inventory detail view - show populated columns/fields button - addressed [here](https://github.com/SEED-platform/seed/issues/1715)

Lastly, I just wanted to clarify that ESPM importing does perform deduplication both **checking against existing records and among the incoming records themselves.** 

Given all that, I think this can be closed.
